tests/helpers.py

```diff
@@ -19,3 +20,45 @@ def load_json(datadir, filename):
     with open(filepath, "r") as file_:
         data = json.load(file_)
     return data
+
+
+def remove_tag_from_marcxml(marcxml, tag):
```

Comment:
Nice helpers! Are these used only for videos or they can be used in the rdm tests as well?